src/mame/sinclair/specnext_divmmc.h

```diff
@@ -0,0 +1,90 @@
+#ifndef MAME_SINCLAIR_SPECNEXT_DIVMMC_H
+#define MAME_SINCLAIR_SPECNEXT_DIVMMC_H
```

Comment:
This file is still missing the license header comment.

Reply:
Let me re-apply couple changes from main PR.